app/src/main/AndroidManifest.xml

```diff
@@ -3,24 +3,39 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="space.taran.arkmemo">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+
+    <uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE"
+        tools:ignore="ScopedStorage" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+
     <application
+        android:name=".App"
         android:allowBackup="true"
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
+        android:requestLegacyExternalStorage="true"
+        tools:replace="android:label"
         android:icon="${appIcon}"
         android:label="${appLabel}"
         android:supportsRtl="true"
         android:theme="@style/Theme.ARKMemo"
         tools:targetApi="31">
```

Comment:
Add this line for ACRA to application scope:
`android:usesCleartextTraffic="true"`

Reply:
Thisis already added in the other branch

Reply:
@ShubertMunthali let's try to reconstruct changes from other branch here, so we would have less conflicts probability.

Reply:
Alright